WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dashboard/posts/PostCardBuilder.kt

```diff
@@ -13,6 +13,8 @@ import org.wordpress.android.ui.mysite.MySiteCardAndItem.Card.DashboardCards.Das
 import org.wordpress.android.ui.mysite.MySiteCardAndItem.Card.DashboardCards.DashboardCard.PostCard.PostCardWithoutPostItems
 import org.wordpress.android.ui.mysite.MySiteCardAndItemBuilderParams.PostCardBuilderParams
 import org.wordpress.android.ui.mysite.MySiteCardAndItemBuilderParams.PostCardBuilderParams.PostItemClickParams
+import org.wordpress.android.ui.mysite.cards.dashboard.posts.PostCardType.CREATE_FIRST
+import org.wordpress.android.ui.mysite.cards.dashboard.posts.PostCardType.CREATE_NEXT
```

Comment:
Suggestion: Can we either 
1. Remove this import and use the full qualifier at all places 
2. Or use the import and only use the Type(CREATE_NEXT, DRAFT etc.) at all places
WDYT 🤔 . I am open to both ways.

Reply:
Used full qualifier here: 7d17f0f